system/jlib/jmetrics.cpp

```diff
@@ -211,7 +238,7 @@ bool MetricsManager::addMetric(const std::shared_ptr<IMetric> &pMetric)
     {
 #ifdef _DEBUG
         // In debug throw an exception so the developer knows there is a regex error
-        throw makeStringExceptionV(MSGAUD_operator, "Metrics manager failed to validate metric name, regex match error, code=%d, what=%s",
+        throw makeStringExceptionV(MSGAUD_operator, "Metrics manager failed to validate metric name/metaData, regex match error, code=%d, what=%s",
```

Comment:
minor, but "name/metaData" could be misleading. I'd go ahead and write out "name, or metaData"